2022-03-16 16:42:41,774 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 16:42:41,777 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 16:42:41,777 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 16:43:02,017 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52493)
2022-03-16 16:43:02,530 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647438182.017801, 'user': {'account_name': 'TestSender2'}}
2022-03-16 16:43:03,053 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647438182.584037, 'account_name': 'TestSender2'}
2022-03-16 16:43:03,556 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647438183.071338, 'user': 'TestSender2'}
2022-03-16 16:43:26,691 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender2', 'to': 'TestSender2', 'time': 1647438206.580317, 'mess_text': 'sdfsdfs'}
2022-03-16 16:43:26,699 INFO server.py Отправлено сообщение пользователю TestSender2 от пользователя TestSender2.
2022-03-16 16:43:37,258 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647438216.81502, 'account_name': 'TestSender2'}
2022-03-16 16:47:00,096 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52531)
2022-03-16 16:47:00,352 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647438420.0956218, 'user': {'account_name': 'TestSender2'}}
2022-03-16 16:49:15,833 INFO server.py Клиент ID: 4 отключился от сервера.
2022-03-16 17:22:42,888 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 17:22:42,893 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 17:22:42,893 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 17:23:00,295 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52644)
2022-03-16 17:23:00,808 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647440580.2955751, 'user': {'account_name': 'TestSender2'}}
2022-03-16 17:23:01,329 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647440580.8548028, 'account_name': 'TestSender2'}
2022-03-16 17:23:01,833 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647440581.3365781, 'user': 'TestSender2'}
2022-03-16 17:23:28,021 INFO server.py Клиент ID: 4 отключился от сервера.
2022-03-16 17:24:36,012 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52650)
2022-03-16 17:24:36,012 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647440676.0125089, 'user': {'account_name': 'TestSender2'}}
2022-03-16 17:25:01,441 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52652)
2022-03-16 17:25:01,441 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647440701.441083, 'user': {'account_name': 'TestSender2'}}
2022-03-16 17:25:18,450 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52653)
2022-03-16 17:25:18,450 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647440718.450627, 'user': {'account_name': 'TestSender22'}}
2022-03-16 17:25:18,969 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647440718.506261, 'account_name': 'TestSender22'}
2022-03-16 17:25:19,474 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647440718.978705, 'user': 'TestSender22'}
2022-03-16 17:25:35,592 INFO server.py Клиент ID: 7 отключился от сервера.
2022-03-16 17:26:10,059 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52655)
2022-03-16 17:26:10,564 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647440770.06026, 'user': {'account_name': 'TestSender22'}}
2022-03-16 17:26:20,465 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52656)
2022-03-16 17:26:20,466 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647440780.465394, 'user': {'account_name': 'TestSender212'}}
2022-03-16 17:26:20,979 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647440780.5056179, 'account_name': 'TestSender212'}
2022-03-16 17:26:21,487 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647440780.996932, 'user': 'TestSender212'}
2022-03-16 17:26:34,580 INFO server.py Клиент ID: 8 отключился от сервера.
2022-03-16 17:45:08,456 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 17:45:08,458 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 17:45:08,458 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 17:45:21,410 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52885)
2022-03-16 17:45:21,410 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647441921.409963, 'user': {'account_name': 'TestSender212'}}
2022-03-16 17:45:21,933 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647441921.46527, 'account_name': 'TestSender212'}
2022-03-16 17:45:22,439 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647441921.952724, 'user': 'TestSender212'}
2022-03-16 17:45:37,037 INFO server.py Клиент ID: 8 отключился от сервера.
2022-03-16 17:54:20,474 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52919)
2022-03-16 17:54:20,762 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647442460.474416, 'user': {'account_name': 'TestSender212'}}
2022-03-16 17:54:47,651 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52922)
2022-03-16 17:54:47,652 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647442487.651768, 'user': {'account_name': 'TestSender212'}}
2022-03-16 17:55:27,492 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52925)
2022-03-16 17:55:27,579 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647442527.492828, 'user': {'account_name': 'TestSender212'}}
2022-03-16 17:56:13,514 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52928)
2022-03-16 17:56:13,600 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647442573.514622, 'user': {'account_name': 'TestSender212'}}
2022-03-16 17:56:49,769 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52931)
2022-03-16 17:56:49,900 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647442609.769556, 'user': {'account_name': 'TestSender212'}}
2022-03-16 17:58:33,197 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52940)
2022-03-16 17:58:33,324 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647442713.197669, 'user': {'account_name': 'TestSender212'}}
2022-03-16 17:59:37,537 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52946)
2022-03-16 17:59:37,622 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647442777.5361862, 'user': {'account_name': 'TestSender212'}}
2022-03-16 18:01:58,170 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52955)
2022-03-16 18:01:58,257 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647442918.170836, 'user': {'account_name': 'TestSender212'}}
2022-03-16 18:03:56,857 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 18:03:56,859 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 18:03:56,859 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 18:06:44,056 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 18:06:44,058 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 18:06:44,058 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 18:06:52,507 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52975)
2022-03-16 18:06:53,019 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647443212.507197, 'user': {'account_name': 'TestSender212'}}
2022-03-16 18:06:53,539 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647443213.072246, 'account_name': 'TestSender212'}
2022-03-16 18:06:54,046 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647443213.559372, 'user': 'TestSender212'}
2022-03-16 18:07:11,662 INFO server.py Клиент ID: 8 отключился от сервера.
2022-03-16 18:14:20,961 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 52997)
2022-03-16 18:14:21,129 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647443660.961759, 'user': {'account_name': 'TestSender212'}}
2022-03-16 18:16:45,036 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53006)
2022-03-16 18:16:45,248 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647443805.035885, 'user': {'account_name': 'TestSender212'}}
2022-03-16 18:21:46,254 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53040)
2022-03-16 18:21:46,477 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647444106.2529862, 'user': {'account_name': 'TestSender212'}}
2022-03-16 18:25:17,241 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53053)
2022-03-16 18:25:17,368 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647444317.241235, 'user': {'account_name': 'TestSender212'}}
2022-03-16 18:26:32,074 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53058)
2022-03-16 18:26:32,453 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647444392.074005, 'user': {'account_name': 'TestSender212'}}
2022-03-16 18:28:47,353 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53069)
2022-03-16 18:28:47,480 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647444527.353153, 'user': {'account_name': 'TestSender212'}}
2022-03-16 18:29:09,944 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53080)
2022-03-16 18:29:52,081 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647444569.832268, 'user': {'account_name': 'TestSender212'}}
2022-03-16 18:30:56,698 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53088)
2022-03-16 18:30:56,865 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647444656.6982222, 'user': {'account_name': 'TestSender2112'}}
2022-03-16 18:30:57,421 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647444656.959999, 'account_name': 'TestSender2112'}
2022-03-16 18:30:57,926 INFO server.py Клиент ID: 9 отключился от сервера.
2022-03-16 18:33:42,140 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53097)
2022-03-16 18:33:42,267 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647444822.140718, 'user': {'account_name': 'TestSender2112'}}
2022-03-16 18:33:54,685 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53102)
2022-03-16 18:33:54,814 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647444834.685327, 'user': {'account_name': 'TestSender21112'}}
2022-03-16 18:33:55,325 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647444834.863791, 'account_name': 'TestSender21112'}
2022-03-16 18:33:55,836 INFO server.py Клиент ID: 10 отключился от сервера.
2022-03-16 18:35:26,736 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53108)
2022-03-16 18:35:26,863 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647444926.7356281, 'user': {'account_name': 'TestSender21112'}}
2022-03-16 18:35:38,539 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53112)
2022-03-16 18:35:38,626 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647444938.539834, 'user': {'account_name': 'TestSender221112'}}
2022-03-16 18:35:39,225 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647444938.7662468, 'account_name': 'TestSender221112'}
2022-03-16 18:35:39,729 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647444939.242213, 'user': 'TestSender221112'}
2022-03-16 18:36:01,875 INFO server.py Клиент ID: 11 отключился от сервера.
2022-03-16 18:39:23,208 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53127)
2022-03-16 18:39:23,334 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647445163.207544, 'user': {'account_name': 'TestSender221112'}}
2022-03-16 18:39:34,510 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53129)
2022-03-16 18:39:34,596 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647445174.51052, 'user': {'account_name': 'TestSender2w21112'}}
2022-03-16 18:39:35,146 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647445174.685256, 'account_name': 'TestSender2w21112'}
2022-03-16 18:39:35,650 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647445175.163163, 'user': 'TestSender2w21112'}
2022-03-16 18:39:51,748 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1647445191.3548229, 'from': 'Guest', 'to': 'TestSender2w21112', 'mess_text': 'sdfdsaf'}
2022-03-16 18:39:51,757 INFO server.py Отправлено сообщение пользователю TestSender2w21112 от пользователя Guest.
2022-03-16 18:40:59,923 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1647445254.9853902, 'from': 'Guest', 'to': 'TestSender2w21112', 'mess_text': 'dsfsdfsd'}
2022-03-16 18:40:59,931 INFO server.py Отправлено сообщение пользователю TestSender2w21112 от пользователя Guest.
2022-03-16 18:41:16,649 INFO server.py Клиент ID: 12 отключился от сервера.
2022-03-16 18:41:17,467 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53135)
2022-03-16 18:41:17,467 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647445277.467525, 'user': {'account_name': 'TestSender2w21112'}}
2022-03-16 18:41:32,135 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53137)
2022-03-16 18:41:32,136 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647445292.135707, 'user': {'account_name': 'TestSender2ww21112'}}
2022-03-16 18:41:32,647 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647445292.179313, 'account_name': 'TestSender2ww21112'}
2022-03-16 18:41:33,152 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647445292.6652029, 'user': 'TestSender2ww21112'}
2022-03-16 18:41:57,351 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1647445316.833815, 'from': 'Guest', 'to': 'TestSender2ww21112', 'mess_text': 'asdfasdfads'}
2022-03-16 18:41:57,359 INFO server.py Отправлено сообщение пользователю TestSender2ww21112 от пользователя Guest.
2022-03-16 18:44:12,761 INFO server.py Клиент ID: 13 отключился от сервера.
2022-03-16 18:45:34,364 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53153)
2022-03-16 18:45:34,451 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647445534.364653, 'user': {'account_name': 'TestSender2ww21112'}}
2022-03-16 18:45:53,155 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53155)
2022-03-16 18:45:53,241 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647445553.1556962, 'user': {'account_name': 'TestSender2www21112'}}
2022-03-16 18:45:53,841 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647445553.378658, 'account_name': 'TestSender2www21112'}
2022-03-16 18:45:54,343 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647445553.8576798, 'user': 'TestSender2www21112'}
2022-03-16 18:46:28,555 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1647445588.102293, 'from': 'Guest', 'to': 'TestSender2www21112', 'mess_text': 'sdgfsfd'}
2022-03-16 18:46:28,556 INFO server.py Отправлено сообщение пользователю TestSender2www21112 от пользователя Guest.
2022-03-16 18:55:32,873 INFO server.py Клиент ID: 14 отключился от сервера.
2022-03-16 18:56:25,851 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53206)
2022-03-16 18:58:12,735 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647446204.6547542, 'user': {'account_name': 'TestSender2wwww21112'}}
2022-03-16 18:58:27,339 INFO server.py Клиент ID: 15 отключился от сервера.
2022-03-16 18:58:36,730 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53218)
2022-03-16 18:58:37,233 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647446316.730699, 'user': {'account_name': 'TestSender2wwww21112'}}
2022-03-16 18:58:48,605 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53222)
2022-03-16 18:58:48,606 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647446328.60559, 'user': {'account_name': 'TestSender2wwwww21112'}}
2022-03-16 18:58:49,119 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647446328.65312, 'account_name': 'TestSender2wwwww21112'}
2022-03-16 18:58:49,622 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647446329.132889, 'user': 'TestSender2wwwww21112'}
2022-03-16 18:59:06,729 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1647446346.620529, 'from': 'Guest', 'to': 'TestSender2wwwww21112', 'mess_text': 'dafsfgrefgewr'}
2022-03-16 18:59:06,738 INFO server.py Отправлено сообщение пользователю TestSender2wwwww21112 от пользователя Guest.
2022-03-16 18:59:32,873 INFO server.py Клиент ID: 16 отключился от сервера.
2022-03-16 19:11:10,669 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53281)
2022-03-16 19:11:10,756 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647447070.6692, 'user': {'account_name': 'TestSender2wwwwww21112'}}
2022-03-16 19:11:17,032 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647447070.893623, 'account_name': 'TestSender2wwwwww21112'}
2022-03-16 19:11:17,542 INFO server.py Клиент ID: 17 отключился от сервера.
2022-03-16 19:14:55,156 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53295)
2022-03-16 19:14:55,243 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647447295.15631, 'user': {'account_name': 'TestSender2wwwwww21112'}}
2022-03-16 19:15:12,959 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53298)
2022-03-16 19:15:13,045 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647447312.959286, 'user': {'account_name': 'TestSender2cwwwwww21112'}}
2022-03-16 19:15:21,718 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647447313.23824, 'account_name': 'TestSender2cwwwwww21112'}
2022-03-16 19:15:22,230 INFO server.py Клиент ID: 18 отключился от сервера.
2022-03-16 19:18:34,707 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53317)
2022-03-16 19:18:55,717 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647447518.986869, 'user': {'account_name': 'TestSender2cqwwwwww21112'}}
2022-03-16 19:19:24,936 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647447564.815603, 'account_name': 'TestSender2cqwwwwww21112'}
2022-03-16 19:22:06,196 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647447720.072383, 'user': 'TestSender2cqwwwwww21112'}
2022-03-16 19:24:10,357 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1647447850.043762, 'from': 'TestSender2cqwwwwww21112', 'to': 'TestSender2cqwwwwww21112', 'mess_text': 'fdsgffdhrtwbyer'}
2022-03-16 19:24:10,365 INFO server.py Отправлено сообщение пользователю TestSender2cqwwwwww21112 от пользователя TestSender2cqwwwwww21112.
2022-03-16 19:31:05,000 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 19:31:05,002 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 19:31:05,002 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 19:32:18,980 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53369)
2022-03-16 19:32:19,151 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647448338.980434, 'user': {'account_name': 'TestSender2acqwwwwww21112'}}
2022-03-16 19:32:19,684 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647448339.22113, 'account_name': 'TestSender2acqwwwwww21112'}
2022-03-16 19:32:20,192 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647448339.699413, 'user': 'TestSender2acqwwwwww21112'}
2022-03-16 19:32:44,352 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1647448364.03235, 'from': 'TestSender2acqwwwwww21112', 'to': 'TestSender2acqwwwwww21112', 'mess_text': 'gtykmjuytensrw'}
2022-03-16 19:32:44,360 INFO server.py Отправлено сообщение пользователю TestSender2acqwwwwww21112 от пользователя TestSender2acqwwwwww21112.
2022-03-16 19:33:53,348 INFO server.py Клиент ID: 20 отключился от сервера.
2022-03-16 19:37:11,952 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53397)
2022-03-16 19:37:12,078 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647448631.951945, 'user': {'account_name': 'TestSender2acqwwwwww21112'}}
2022-03-16 19:44:21,641 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53429)
2022-03-16 19:44:21,778 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647449061.641098, 'user': {'account_name': 'TestSender2acqwwwwww21112'}}
2022-03-16 19:49:01,729 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции main().
2022-03-16 19:49:01,731 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 19:49:01,731 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 19:49:30,687 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53467)
2022-03-16 19:51:22,160 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции main().
2022-03-16 19:51:22,162 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 19:51:22,162 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 19:51:40,916 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53482)
2022-03-16 19:54:04,565 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647449500.9162612, 'user': {'account_name': 'TestSender2acqwwwwww21112'}}
2022-03-16 20:14:24,447 INFO server.py Клиент ID: 20 отключился от сервера.
2022-03-16 20:15:59,912 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53572)
2022-03-16 20:16:29,368 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647450959.9124181, 'user': {'account_name': 'TestSender2acqwwwwww21112'}}
2022-03-16 20:18:53,337 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53580)
2022-03-16 20:27:42,606 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647451133.337522, 'user': {'account_name': 'TestSender2acqwwwwww21112'}}
2022-03-16 20:29:21,771 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 20:29:21,772 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 20:29:21,772 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 20:30:34,291 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 20:30:34,292 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 20:30:34,292 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 20:31:18,128 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53637)
2022-03-16 20:31:18,128 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647451878.1279519, 'user': {'account_name': 'TestSender2acqwwwwww21112'}}
2022-03-16 20:31:18,650 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647451878.182076, 'account_name': 'TestSender2acqwwwwww21112'}
2022-03-16 20:31:19,158 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647451878.6704938, 'user': 'TestSender2acqwwwwww21112'}
2022-03-16 20:31:51,368 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1647451910.9165728, 'from': 'TestSender2acqwwwwww21112', 'to': 'TestSender2acqwwwwww21112', 'mess_text': 'sdgdsfagafsdg'}
2022-03-16 20:31:51,369 INFO server.py Отправлено сообщение пользователю TestSender2acqwwwwww21112 от пользователя TestSender2acqwwwwww21112.
2022-03-16 20:32:24,600 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647451944.412695, 'user': 'TestSender2acqwwwwww21112', 'account_name': 'TestSender4'}
2022-03-16 20:33:05,511 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647451985.37515, 'account_name': 'TestSender2acqwwwwww21112'}
2022-03-16 20:33:33,195 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 53646)
2022-03-16 20:33:33,699 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647452013.195808, 'user': {'account_name': 'TestSender2'}}
2022-03-16 20:33:34,215 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647452013.7408729, 'account_name': 'TestSender2'}
2022-03-16 20:33:34,721 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647452014.238252, 'user': 'TestSender2'}
2022-03-16 20:33:52,852 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'time': 1647452032.502934, 'from': 'TestSender2', 'to': 'TestSender2', 'mess_text': 'sgvwbnguqerhf;'}
2022-03-16 20:33:52,852 INFO server.py Отправлено сообщение пользователю TestSender2 от пользователя TestSender2.
2022-03-16 20:34:00,391 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647452039.9748921, 'account_name': 'TestSender2'}
